sdl_android_lib/src/com/smartdevicelink/util/SdlDataTypeConverter.java

```diff
@@ -0,0 +1,18 @@
+package com.smartdevicelink.util;
+
+public class SdlDataTypeConverter {
+	
+	public static Double ObjectToDouble(Object obj) {
```

Comment:
Also, for completeness (I realize this isn't relevant to the case we're addressing), this method might want to allow conversions from floats and possibly strings as well.  If a float like `3.14159f` is passed in, the method will return null.  Similarly, if a string like `"3.14159"` is passed in, the method will return null.  Technically, both of these values could be converted to a double.

If someone wants to use this method sometime in the future for some other purpose, it might have undesired behaviors in these cases.